cvat/apps/auto_annotation/README.md

```diff
@@ -0,0 +1,114 @@
+Supported only DLDT framework from OpenVINO&trade; toolkit. If you woold like to annotate task with custom model please convert it to the
+intermediate representation (IR) format via model optimizer tool.
+See [OpenVINO documentation](https://software.intel.com/en-us/articles/OpenVINO-InferEngine) for details.
+
+### Usage
+To annotate task with a custom model you need prepare 4 files:
+1. __Model config__ - a text file that contains network configuration. The following file extension is expected:
+   * __*.xml__
+1. __Model weights__ - a binary file that contains trained weights. The following file extension is expected:
+   * __*.bin__
+1. __Label map__ - simple json file that contains the `label_map` dictionary like object with string values for label numbers.
+Please note values in `label_map` should be exactly equal to the labels wich task was created, otherwise it will be ignored.
+  Example:
+    ```json
+    {
+      "label_map": {
+        "0": "background",
+        "1": "aeroplane",
+        "2": "bicycle",
+        "3": "bird",
+        "4": "boat",
+        "5": "bottle",
+        "6": "bus",
+        "7": "car",
+        "8": "cat",
+        "9": "chair",
+        "10": "cow",
+        "11": "diningtable",
+        "12": "dog",
+        "13": "horse",
+        "14": "motorbike",
+        "15": "person",
+        "16": "pottedplant",
+        "17": "sheep",
+        "18": "sofa",
+        "19": "train",
+        "20": "tvmonitor"
+      }
+    }
+    ```
+1. __Interpretation script__ - python file that used to convert output results from net to CVAT format. This code running inside restricted environment.
+List of builtins functions that available to use:
+   * __str__
+   * __int__
+   * __float__
+   * __max__
+   * __min__
+   * __range__
+
+   Also two variables are available in scope:
+   * **detections** list with detection results(see description below)
+   * **results** dictionary where convertation results shoud be added (see examples below for details)
+
+   `detection` is a python's list of dictionaries that represent detections for each frame of task with folloing keys:
+   * __frame_id__ - frame number
+   * __frame_height__ - frame height
+   * __frame_width__ - frame width
+   * __detections__ - output np.ndarray (See [ExecutableNetwork.infer](https://software.intel.com/en-us/articles/OpenVINO-InferEngine#inpage-nav-11-6-3) for details).
+
+   `results` is dictionary with structure:
+   ```python
+    {
+      "boxes": [],
+      "polygons": [],
+      "polylines": [],
+      "points": [],
+      "box_paths": [],
+      "polygon_paths": [],
+      "polyline_paths": [],
+      "points_paths": [],
+    }
+    ```
+
+    Example for SSD based network:
+    ```python
+    def process_results(detections, results):
+    def clip(value):
+        return max(min(1.0, value), 0.0)
+
+    boxes = results['boxes']
+
+    for frame_results in detections:
```

Comment:
How the function will look for the model: file:///opt/intel/computer_vision_sdk_2018.5.445/deployment_tools/intel_models/person-detection-retail-0002/description/person-detection-retail-0002.html?